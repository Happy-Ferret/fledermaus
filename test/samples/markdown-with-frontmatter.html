---
layout: post
lang: en
title: 'Read less tech books'
date: Feb 20, 2014
tags:
  - thoughts
  - something
---

<p>Vladimir Starkov <a href="http://vstarkov.com/monthbook/">suggests</a> to read at least two technical books a month.</p>
<p>Itâ€™s very important to be up-to-date with latest trends, especially in so fast changing industry as front-end development. But there are so many interesting subjects besides  programming languages and new frameworks.</p>
