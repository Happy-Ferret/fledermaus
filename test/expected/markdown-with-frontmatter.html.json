{
    "layout": "post",
    "lang": "en",
    "title": "Read less tech books",
    "date": "Feb 20, 2014",
    "tags": ["thoughts", "something"],
    "sourcePath": "test/samples/markdown-with-frontmatter.html",
    "url": "/test/samples/markdown-with-frontmatter",
    "content": "\n<p>Vladimir Starkov <a href=\"http://vstarkov.com/monthbook/\">suggests</a> to read at least two technical books a month.</p>\n<p>Itâ€™s very important to be up-to-date with latest trends, especially in so fast changing industry as front-end development. But there are so many interesting subjects besides  programming languages and new frameworks.</p>\n"
}
